Description: Fix some spelling errors
 Fix spelling errors in slurmdbd_defs.c
Author: Gennaro Oliva <oliva.g@na.icar.cnr.it>
Forwarded: https://bugs.schedmd.com/show_bug.cgi?id=3111
Last-Update: 2016-09-30

--- slurm-llnl-16.5.4.orig/src/common/slurmdbd_defs.c
+++ slurm-llnl-16.5.4/src/common/slurmdbd_defs.c
@@ -980,9 +980,9 @@ extern slurmdbd_msg_type_t str_2_slurmdb
 		return DBD_STEP_START;
 	} else if (!xstrcasecmp(msg_type, "Get Jobs Conditional")) {
 		return DBD_GET_JOBS_COND;
-	} else if (!xstrcasecmp(msg_type, "Get Transations")) {
+	} else if (!xstrcasecmp(msg_type, "Get Transactions")) {
 		return DBD_GET_TXN;
-	} else if (!xstrcasecmp(msg_type, "Got Transations")) {
+	} else if (!xstrcasecmp(msg_type, "Got Transactions")) {
 		return DBD_GOT_TXN;
 	} else if (!xstrcasecmp(msg_type, "Add QOS")) {
 		return DBD_ADD_QOS;
@@ -1400,13 +1400,13 @@ extern char *slurmdbd_msg_type_2_str(slu
 		if (get_enum) {
 			return "DBD_GET_TXN";
 		} else
-			return "Get Transations";
+			return "Get Transactions";
 		break;
 	case DBD_GOT_TXN:
 		if (get_enum) {
 			return "DBD_GOT_TXN";
 		} else
-			return "Got Transations";
+			return "Got Transactions";
 		break;
 	case DBD_ADD_QOS:
 		if (get_enum) {
